langs/yuescript/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:bookworm-slim AS builder
```

Comment:
As of now, there isn't the slightest chance to build the source on the Alpine 320 base. As such, we'll have to manage with Debian.